### ISSUE TITLE
Core Updater failing to download index

### ISSUE THREAD

**Issue by Ryunam** [libretro/RetroArch#6887]
## Description

When trying to use the Core Updater option within RA, an error message currently appears stating "Task failed: Downloading index", resulting in an empty list.

### Expected behavior

Core Updater should display the list of available cores, as per the usual behavior exhibited by that option.

### Version/Commit

The issue is occurring with the latest code as of the time of this message.

- RetroArch: 1.7.3, commit 9d0c0b77eee7aecc8568e03ec3f8fbc1324dfd1d

### Environment information

- OS: Windows 10 x64, build 1709
- Compiler: Mingw64 (but it also happens with pre-compiled nightlies from the Buildbot)

**Comment by RobLoach** [comment]
The issue template has some information that could be helpful...

```
### Version/Commit
You can find this information under Information/System Information

- RetroArch: [version/commit]

### Environment information

- OS: [The operating system you're running]
- Compiler: [In case you are running local builds]
```

**Comment by Ryunam** [comment]
Sorry @Robloach, I have now added all the required information to my first post. In the meantime, I managed to figure out what was causing this.

The "core_updater_buildbot_url" option within my retroarch.cfg file was pointing to "http://buildbot.libretro.com/nightly/win-x86_64/latest/", as opposed to ""http://buildbot.libretro.com/nightly/windows/x86_64/latest/". Replacing it with the latter fixed the problem entirely.

Was this changed recently? In any case, as far as I'm concerned, this issue is solved now.
